### PR TITLE
Update it.po

### PR DESCRIPTION
**Pull request by albanobattistella**

### CODE REVIEW

File: po/it.po [hunk 1]
```diff
@@ -1492,6 +1494,9 @@ msgid ""
 "%s\n"
 "Orientation: [%f, %f, %f], %.1f\n"
 msgstr ""
```
Comment (375gnu) [inline po/it.po:1496]: please remove this line

**Comment by 375gnu** [comment]
@albanobattistella could you merge your commits into a single one?

**Comment by albanobattistella** [comment]
> @albanobattistella could you merge your commits into a single one?

Hi,
can you let me know the procedure?Thanks.

**Comment by eyvallah** [comment]
@albanobattistella I would usually use git rebase -i to do this

try `git rebase -i HEAD^^` then replace every `pick` with `squash` at the start of lines except for your most recent commit, quit the editor and it should prompt you to write commit message for the squashed commit.

File: po/it.po [hunk 2]
```diff
@@ -5359,7 +5382,7 @@ msgstr "Caricamento di: "
 #: ../src/celestia/win32/winmain.cpp:3364 ../src/celutil/util.cpp:69
 #: ../src/celutil/util.cpp:73
 msgid "LANGUAGE"
-msgstr "it"
+msgstr "LINGUA"
```
Comment (375gnu) [inline po/it.po:5385]: this shoud be 'it', short  language code

**Comment by 375gnu** [comment]
this is superseded by #749

**Pull request closed without merging**